test-expected/sequelize/dumper-output/default-values.expected.js

```diff
@@ -5,47 +5,85 @@ module.exports = (sequelize, DataTypes) => {
   // This section contains the fields of your model, mapped to your table's columns.
   // Learn more here: https://docs.forestadmin.com/documentation/v/v6/reference-guide/models/enrich-your-models#declaring-a-new-field-in-a-model
   const DefaultValues = sequelize.define('defaultValues', {
-    noDefault: {
-      type: DataTypes.STRING,
-    },
-    arrayDefault: {
-      type: DataTypes.ARRAY(DataTypes.INTEGER),
-      defaultValue: [],
+    boolNull: {
+      type: DataTypes.BOOLEAN,
+      field: 'bool_null',
     },
-    booleanDefault: {
+    boolCst: {
```

Comment:
[WOrding] We don't do shortcut in namings. Plus keep the old naming it's just increasing the diff 😅

Reply:
Responded in another comment